packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1,11 +1,28 @@
 import { cloudformation as applicationautoscaling } from '@aws-cdk/aws-applicationautoscaling';
 import { Role } from '@aws-cdk/aws-iam';
+import iam = require('@aws-cdk/aws-iam');
 import { Construct, PolicyStatement, PolicyStatementEffect, ServicePrincipal } from '@aws-cdk/cdk';
 import { cloudformation as dynamodb } from './dynamodb.generated';
 
 const HASH_KEY_TYPE = 'HASH';
 const RANGE_KEY_TYPE = 'RANGE';
 
+const READ_DATA_ACTIONS = [
```

Comment:
Doesn't this belong to a different PR?

Reply:
ha! i wondered where these changes disappeared. Yes, of course. This needs to go to a separate PR.